quetz/tasks/mirror.py

```diff
@@ -235,38 +261,24 @@ def handle_repodata_package(
         total_size += size
         file.file.seek(0)
 
-    dao.assert_size_limits(channel_name, total_size)
+    # create package in database
```

Comment:
Should this be uncommented?

Reply:
The problem is that if we create packages already create packages from the channel metadata here (which seems to be some kind of.. shortcut?) than we have problems filtering them out later.

some solutions
1. call this code if no includelist/excludelist is set -> then we already sync everything
2. do the filtering based on includelist/excludelist here as well -> meh

Reply:
I've implemented no. 1 for now